cranelift/filetests/filetests/isa/riscv64/fcmp.clif

```diff
@@ -18,8 +17,12 @@ block1:
 ;   fmv.d.x ft1,t1
 ;   li a2,0
 ;   fmv.d.x ft5,a2
-;   fle.d a5,ft5,ft1
-;   bne a5,zero,taken(label1),not_taken(label2)
+;   feq.d a5,ft1,ft1
+;   feq.d a7,ft5,ft5
+;   and t4,a5,a7
+;   fle.d t1,ft5,ft1
+;   and a0,t4,t1
+;   beq a0,zero,taken(label1),not_taken(label2)
```

Comment:
This ballooned a bit because it's now testing `ult`, rather than the negation of `ult` which is just `ge`. We could get back to the old implementation by swapping the branches and inverting the condition code.

Reply:
I fixed this separately in #5636. I realized we could invert the condition code and branches to avoid the cost of the unordered comparison in the riscv64 backend :+1: